### ISSUE TITLE
Passing arguments to node script via `--` doesn't work

### ISSUE THREAD

**Issue by billyvg** [Unitech/PM2#1335]
`pm2 start runner.js -- test`

When printing `process.argv`pm2 logs shows:

```
[ 'node',
'/Users/billy/Development/woopra/woopra-apprunner/runner.js',
'[Circular ~.args]' ]
```

```
woopra-apprunner git/master*
❯ node -v
v0.12.2

woopra-apprunner git/master*
❯ pm2 -v
0.12.14
```

**Comment by Unitech** [comment]
I just tried this script: https://github.com/Unitech/PM2/blob/master/examples/args.js

```
$ pm2 start examples/args.js -- hey hay hot
```

And I get this output:

```
args-0 (out): HERE ARE MY ARGS !!! =  [ 'node',
args-0 (out):   '/home/shant/keymetrics/pm2/examples/args.js',
args-0 (out):   'hey',
args-0 (out):   'hay',
args-0 (out):   'hot' ]
```

Node 0.12.2, pm2 0.12.16

**Comment by billyvg** [comment]
It seems to be fixed in 0.12.16, thanks
